plugins/BEdita/Core/src/Job/ServiceRegistry.php

```diff
@@ -49,11 +49,15 @@ public static function get($name)
             return static::$instances[$name];
         }
 
-        $className = Inflector::camelize($name);
-        $fullClassName = App::className($className, 'Job/Service', 'Service');
-        if ($fullClassName === false && strpos($className, '.') === false) {
-            $fullClassName = App::className('BEdita/Core.' . $className, 'Job/Service', 'Service');
+        $plugin = 'BEdita/Core.';
+        if (strpos($name, '.') !== false) {
+            list($plugin, $name) = explode('.', $name);
+            $plugin = Inflector::camelize($plugin) . '.';
         }
+
+        $className = $plugin . Inflector::camelize($name);
+        $fullClassName = App::className($className, 'Job/Service', 'Service');
```

Comment:
Since BE4 has a plugin architecture it's not recommended to do anything in the `app` - the right way to customize something is via plugins.